spring-cloud-modules/spring-cloud-data-flow/spring-cloud-data-flow-stream-processing/data-flow-server/pom.xml

```diff
@@ -62,13 +49,22 @@
             <version>${byte-buddy.version}</version>
             <scope>test</scope>
         </dependency>
+         <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy</artifactId>
+            <version>${groovy.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.session</groupId>
+            <artifactId>spring-session-data-redis</artifactId>
+        </dependency>
     </dependencies>
 
     <properties>
```

Comment:
spring-cloud-dependencies.version was removed as not having any effect (like above removed parts)
Hibernate was upgraded to minimum needed version for JDK 17 (5.6)
spring-cloud-dataflow-dependencies and spring-boot were upgraded to related versions.